src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -3306,11 +3306,47 @@ static void Main(string[] args)
                 Diagnostic(RudeEditKind.StackAllocUpdate, "stackalloc", FeaturesResources.method));
         }
 
+        [Theory]
+        [InlineData("stackalloc int[3]")]
+        [InlineData("stackalloc int[3] { 1, 2, 3 }")]
+        [InlineData("stackalloc int[] { 1, 2, 3 }")]
+        [InlineData("stackalloc[] { 1, 2, 3 }")]
+        public void MethodUpdate_UpdateStackAlloc2(string stackallocDecl)
```

Comment:
Is there a follow-up issue tracking making this (and other) edits non-rude?

Reply:
No, we don't track those in general. This one would require CLR fix.